react-time-track/src/utils/calculateWorkDays.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * function parseDate(input)
+ * given a date input, it checks if it's a string or a date object and only returns the date as an object
+ * @param {string||object} input
+ */
+function parseDate(input) {
+  if (typeof input === "object") input = dateToString(input);
+  let parts = input.match(/(\d+)/g);
+  return new Date(parts[0], parts[1] - 1, parts[2]);
+}
```

Comment:
Hi, I get the point about creating this function to get the date, I think you could use other functions which belong to the object date to do the same, but I notice it has logical only in this file, so it is better to understand the code.

Reply:
You are right, this function gets a string and turns into an array. A way to clean the input. It might be useful in future versions of the project.